pipeline-templates/job/deploy.yml

```diff
@@ -65,7 +65,6 @@ jobs:
               EASMessageServiceBusString: $(EASMessageServiceBusString)
               EmployerAccountsApimSubscriptionKey: $(EmployerAccountsApimSubscriptionKey)
               EmployerSharedServiceBusString: $(EmployerSharedServiceBusString)
-              EventsApiToken: $(EventsApiToken)
```

Comment:
Removing this will cause the pipeline to fail as it will be looking at the master copy of the employer-config which still has the environment variable called EventsApiToken.

https://github.com/SkillsFundingAgency/das-employer-config/blob/master/das-employer-accounts/SFA.DAS.EmployerAccounts.schema.json#L158.